### ISSUE TITLE
iTunes style cover flow style

### ISSUE THREAD

**Issue by iosscripts** [iosscripts/iosslider#103]
Modify script to support iTunes cover flow style animations, tilting prev/next slides and centering current.

**Comment by Pixelous** [comment]
Hi Marc!

What does this post mean? You want to make iTunes style for iosSlider? If so would be really cool!

**Comment by marcwhitbread** [comment]
Yep. I have some ideas for a plugin extension which would change the slider style to have the previous and next slides tilt in. Ideally, it would look exactly like the new iTunes store slider... but touch compatible!

Obviously all the old features would be in there, but snapSlideCenter and snapToChildren would need to be enabled. The ability to customize the tilt would be added.

The drawback would be that this would only work on CSS3 3d transform enabled browsers :S.

**Comment by Pixelous** [comment]
Sounds very good! But iTunes store slider is touch compatible actually. Have tested it on iPad this moment. Any way looking forward to your plugin extension! How long to wait?

**Comment by marcwhitbread** [comment]
Oh is it? My iPad doesn't seem to have the updated version on it. Not a big deal either way. A jQuery plugin version would still be valuable.

It will take a while to build and test probably. I am currently rebuilding iosscripts.com to make way for additional plugins... then I might do that next.

I will let you know when I have something to look at though :).
